apps/nextjs-website/src/components/Editorial.tsx

```diff
@@ -16,6 +15,7 @@ const makeEditorialProps = ({
   storeButtons,
   ...rest
 }: EditorialSection): EditorialProps => ({
+  theme,
```

Comment:
Remove this, together with the `theme` at line 10, since we are not parsing the value in any way and can simply pass it inside of `...rest`